src/main/java/uk/gov/hmcts/cmc/claimstore/models/CountyCourtJudgment.java

```diff
@@ -23,8 +24,9 @@
     @NotNull
     private final TheirDetails defendant;
 
-    @Min(0)
-    private double paidAmount;
+    @Valid
+    @Money
+    private BigDecimal paidAmount;
```

Comment:
I think this should have a minimum on it and `@Money` shouldn't have the min constraint on it

Reply:
Hi Tim I've talked with Patryk about that and at the moment all places where we use money require the value to be at least `0.01`. I agree that these two should be separate but not sure if it's worth it if it would be done for all of them.